IOS/SwiftSeedProject/SwiftSeedProject/Interfaces/Components/MSButton.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  MSButton.swift
+//  SwiftSeedProject
+//
+//  Created by Mugu on 3/20/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import UIKit
+
+@IBDesignable public class MSButton: UIButton {
+
+    @IBInspectable var borderColor: UIColor = UIColor.white {
+        didSet {
+            layer.borderColor = borderColor.cgColor
+            setNeedsLayout()
+        }
+    }
+
+    @IBInspectable var borderWidth: CGFloat = 2.0 {
+        didSet {
+            layer.borderWidth = borderWidth
+            setNeedsLayout()
+        }
+    }
+    
+    @IBInspectable var rounded: Bool = true{
```

Comment:
Minor, edge-case but for the sake of 'correctness'..

If someone sets rounded to true and then to false, it won't restore corner radius.

Maybe you should do something like 

`layer.cornerRadius = rounded ? radius * bounds.size.width : 0`